RundeckAutomation/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Usually this file goes in the repo root and we use a `CopyFile` build task in whiskey.yml to copy it into the module root before packaging. That should probably be in the default whiskey.yml file.

Same with the `NOTICE` and `README.md` files: keep them in the root, and copy them into the module root before packaging.

Also, the NOTICE file is empty.

Reply:
These should be in the right locations now and populated.